app/routes/teams/team.js

```diff
@@ -0,0 +1,11 @@
+import Route from '@ember/routing/route';
+import { ALL_TEAMS } from '../teams';
+
+export default class TeamsTeamRoute extends Route {
+  model({ teamId }) {
+    const matches = ALL_TEAMS.filter(
```

Comment:
Seems like you really want `.find` here, no?

Reply:
is this polyfilled for ie11 by default?

Reply:
No, it wouldn't be. If we are targeting IE11 we should stick with `.filter()[0]`.

Reply:
I think I'll leave it as-is, so that the workshop is operable on all of the browsers ember supports by default